pkg/html/html.go

```diff
@@ -396,6 +396,12 @@ func summaryTopFailingTests(topFailingTestsWithoutBug, topFailingTestsWithBug []
 	`
 
 	for _, test := range topFailingTestsWithoutBug {
+		// if we only have one failure, don't show it on the glass.  Keep it in the actual data so we can choose how to handle it,
+		// but don't bother creating the noise in the UI for a one-off/long tail.
+		if (test.Failures + test.Flakes) == 1 {
+			continue
+		}
```

Comment:
is the minTestRuns not already filtering this out?

if you only have 1 failure and you have at least 10 runs, you're passing at 90% and i wouldn't expect you to be a top failure anyway.